dockers/docker-fpm-frr/base_image_files/TSA

```diff
@@ -1,3 +1,14 @@
 #!/bin/bash
 
+# toggle the mux to standby if dualtor and any mux active
+[[ "$(sonic-cfggen -d -v DEVICE_METADATA.localhost.subtype | tr [:upper:] [:lower:])" == *"dualtor"* ]]
+is_dualtor=$?
+[[ $(show mux status | grep active | wc -l) > 0 ]]
+any_active=$?
+
+if [[ (${is_dualtor} -eq 0) && (${any_active} -eq 0) ]];
+then
```

Comment:
Add a log stating the toggle is happenning?

Reply:
Added, thanks~

Reply:
Did you test the code? I think you introduced syntax error.

Reply:
Yes, tested with `set -x`
```
++ tr '[:upper:]' '[:lower:]'
++ sonic-cfggen -d -v DEVICE_METADATA.localhost.subtype
+ [[ dualtor == *\d\u\a\l\t\o\r* ]]
++ show mux status
++ wc -l
++ grep active
+ [[ 24 > 0 ]]
+ logger -t TSA -p user.info 'Toggle all mux mode to standby'
+ sudo config mux mode standby all
```

Reply:
Yes, tested with `set -x`
1. on a dualtor
```
++ sonic-cfggen -d -v DEVICE_METADATA.localhost.subtype
++ tr '[:upper:]' '[:lower:]'
+ [[ dualtor == *\d\u\a\l\t\o\r* ]]
++ grep active
++ wc -l
++ show mux status
+ [[ 24 > 0 ]]
+ logger -t TSA -p user.info 'Toggle all mux mode to standby'
+ sudo config mux mode standby all
```
2. on a t0
```
++ sonic-cfggen -d -v DEVICE_METADATA.localhost.subtype
++ tr '[:upper:]' '[:lower:]'
+ [[ '' == *\d\u\a\l\t\o\r* ]]
```